docs/overview/bloc-did-method.md

```diff
@@ -8,99 +8,181 @@ The `did:bloc` DID method allows groups of independent organizations to share cu
 
     .well-known/did-bloc/
-        consortium/
-            conf.json
-        stakeholders/
+        [domain].json
         history/
-    
-##### The `consortium` directory
-This directory contains the current consortium configuration, according to this particular server.
-
-###### `conf.json`
-The consortium config contains the following:
-- The domain name of the consortium
-- Consortium policy configuration settings
-- A list of stakeholders - containing, for each stakeholder:
-  - The web domain where its configuration can be found
-  - A local hashlink to the cached stakeholder configuration
-- A list of detached JWS signatures, from the stakeholders, on the config file
-- A local hashlink to the previous version of this config file
+
+`[domain].json`:
+This file contains the current consortium configuration, according to this particular server.
+
+`history`: contains the history database.
+
+##### Consortium Config Files
+A consortium config file is a concatenated JSON file:
+The first element is a JSON object containing:
+  - The domain name of the consortium
+  - Consortium policy configuration settings
+  - A list of stakeholders - containing, for each stakeholder:
+    - The web domain where its configuration can be found
+    - The did:bloc DID of the stakeholder, with the associated DID doc in Sidetree on the consortium ledger
+  - The hash of the previous version of this config file
+After that, in order:
+  - A list of detached JWS signatures, from the stakeholders, on the config file
+  - A list of cached copies of the stakeholder config files
 
 ###### Consortium Endorsement Signatures
 Stakeholders endorse the contents of a consortium configuration file by signing the contents.
 
-Verification: The signatures array must be removed from the config file and the config file must be canonicalized (TODO: define how) before the signatures are verified against the file.
+First the file has appended data removed.
+- The signatures array is removed
+- The stakeholder config cache is removed
 
-##### Stakeholder Configurations
-Within `stakeholders/` are files for stakeholder configs.
+Verification: The signatures array must be removed from the config file and the config file must be canonicalized before the signatures are verified against the file.
 
-There must be at least one stakeholder config file here, for the consortium domain to be usable for resolving DIDs.
-
-Each of these files is named `[hash].json`, where `hash` is the SHA-256 hash of the contents of the file. This hash value is the value used in the consortium config to link to this stakeholder config.
+##### Stakeholder Config Files
+Each of these files is named `[domain].json`, where `[domain]` is the URL domain, owned by the stakeholder organization, where you can find the canonical copy of the stakeholder's configuration.
 
 A stakeholder configuration file contains:
 - The stakeholder's domain
+- The stakeholder's DID (did:bloc)
```

Comment:
Should we also make use of https://identity.foundation/specs/did-configuration/

@llorllale @Moopli

Reply:
I think so, yes.

I hadn't looked at [`did:web`](https://w3c-ccg.github.io/did-method-web/) in a while and I'm now realizing it looks like a rough draft (with last commit 8 months ago). 

This DIF spec looks more polished. *This* is the one that doesn't embed the DID documents into `.well-known`, but `did:web` does. But that's OK because we need all member DIDs registered in the sidetree nodes.